### DISCUSSION TITLE
Communication with Libre VNA

### DISCUSSION THREAD

**Discussion by ndagijif** [jankae/LibreVNA#197] [General]
My libre VNAs can not run on my computers with Windows 10. We checked all possible reasons like firewall, or admin rights on computers, the "meas" on calibration steps does not respond.

Have you an idea of how to troubleshoot this USB communication issue ?

Note : on computers with Windows 10 "family" there is no problem.

**Comment by jankae** [comment]
Standard question: which version of the GUI and firmware are you using?

> the "meas" on calibration steps does not respond.

Can you add some details here please? My best guess is that you mean a SCPI command.

**Comment by ndagijif** [comment]
Thank you very much for your anwser. 
Here is the sofware we use : 


Also when I lunch the cal section here is the message : the call never starts ! 

We are unable fo find out the problem occuring on many computers. 
Regards 

Fabien NDAGIJIMANA 
Professeur Université de Grenoble 
IUT1 - Chef dépt Génie Electrique et Informatique Industrielle 
Laboratoire G2Elab (UMR 5269) / Plateforme RF&CEM Pheline 

Tel 06 19 07 01 85 


De: "jankae" ***@***.***> 
À: "jankae/LibreVNA" ***@***.***> 
Cc: "'fabien ndagijimana" ***@***.***>, "Author" ***@***.***> 
Envoyé: Vendredi 3 Mars 2023 19:43:55 
Objet: Re: [jankae/LibreVNA] Communication with Libre VNA (Discussion #197) 






Standard question: which version of the GUI and firmware are you using? 



the "meas" on calibration steps does not respond. 



Can you add some details here please? My best guess is that you mean a SCPI command.

**Comment by jankae** [comment]
Sorry, I don't see any program version in your message. Did you try to attach screenshots? Not sure how that is handled by Github when you reply by mail. Please try responding through the Github user interface.

**Comment by ndagijif** [comment]
Now I have found : the version we use is 1.3 

We try to update with 1.4 or 1.5 but we have no idea of the issue with USB connection. 

Cordialement 

Fabien NDAGIJIMANA 
Professeur Université de Grenoble 
IUT1 - Chef dépt Génie Electrique et Informatique Industrielle 
Laboratoire G2Elab (UMR 5269) / Plateforme RF&CEM Pheline 

Tel 06 19 07 01 85 


De: "jankae" ***@***.***> 
À: "jankae/LibreVNA" ***@***.***> 
Cc: "'fabien ndagijimana" ***@***.***>, "Author" ***@***.***> 
Envoyé: Mercredi 8 Mars 2023 19:44:09 
Objet: Re: [jankae/LibreVNA] Communication with Libre VNA (Discussion #197) 





Sorry, I don't see any program version in your message. Did you try to attach screenshots? Not sure how that is handled by Github when you reply by mail. Please try responding through the Github user interface.

**Comment by jankae** [comment]
> We try to update with 1.4 or 1.5 but we have no idea of the issue with USB connection.

Yes please update, 1.3.0 is rather old. I am not sure what you mean by "issue with USB connection". Is this still the same issue you originally mentioned? Is your GUI able to find the device at all?

**Comment by hungmks** [comment]
I have the similar issue. The GUI cannot find the device.
The booting LED is not light :(
https://github.com/jankae/LibreVNA/issues/238
